kubevirtci/kubevirtci.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+_action=$1
+_pwd=$(pwd)
+
+function kubevirtci::registry() {
+  echo "localhost:$("./cluster-up/cli.sh" ports registry 2>/dev/null)"/kubevirt/ssp-operator
+}
+
+source kubevirtci/kubevirt.sh
+kubevirt::install
+cd "$(kubevirt::path)"
+
+case ${_action} in
+  "up")
+    make cluster-up && make cluster-sync
```

Comment:
This is not only bringing up a fresh cluster, but also syncing kubevirt to it. But what e.g. about CDI? It would still be missing. It might make more sense to bring up a fresh cluster only and then use https://github.com/kubevirt/ssp-operator/blob/master/automation/common/deploy-kubevirt-and-cdi.sh.

Reply:
CDI is already pulled in by default by `kubevirt/kubevirt` and `make cluster-up`.

Reply:
Ok fair point, but what about deploying the latest kubevirt release instead of rebuilding master? It would save some resources and time I guess.

https://github.com/kubevirt/containerdisks/blob/c625aa253621670f273dae9a1a6d636fee239487/hack/kubevirtci.sh#L25